skimage/feature/util.py

```diff
@@ -96,18 +100,22 @@ def plot_matches(ax, image1, image2, keypoints1, keypoints2, matches,
         new_image2[:image2.shape[0], :image2.shape[1]] = image2
         image2 = new_image2
 
-    image = np.concatenate([image1, image2], axis=1)
-
-    offset = image1.shape
+    offset = np.array(image1.shape)
+    if left_right is True:
```

Comment:
As a side comment, the Pythonic way to use bool arguments is, for example, `if left_right:`. In which case a different renaming is suggested: `if horizontal_alignment:`. But as I mentioned before, I would prefer:

```python
if alignment == 'horizontal':
    ...
elif alignment == 'vertical':
    ...
else:
    raise ValueError('incorrect value for alignment: %s' % alignment)
```

Reply:
Thanks for the tip.

I agree that `is True` is verbose.

Your suggestion keeps the best of both :D